public/_translations/en.json

```diff
@@ -158,5 +158,283 @@
+  "MX": "Mexico",
+  "MY": "Malaysia",
+  "MZ": "Mozambique",
+  "NA": "Namibia",
+  "NC": "New Caledonia",
+  "NE": "Niger",
+  "NF": "Norfolk Island",
+  "NG": "Nigeria",
+  "NI": "Nicaragua",
+  "NL": "Netherlands",
+  "NO": "Norway",
+  "NP": "Nepal",
+  "NR": "Nauru",
+  "NU": "Niue",
+  "NZ": "New Zealand",
+  "OM": "Oman",
+  "PA": "Panama",
+  "PE": "Peru",
+  "PF": "French Polynesia",
+  "PG": "Papua New Guinea",
+  "PH": "Philippines",
+  "PK": "Pakistan",
+  "PL": "Poland",
+  "PM": "Saint Pierre and Miquelon",
+  "PN": "Pitcairn",
+  "PR": "Puerto Rico",
+  "PS": "Palestine, State of",
+  "PT": "Portugal",
+  "PW": "Palau",
+  "PY": "Paraguay",
+  "QA": "Qatar",
+  "RE": "Réunion",
+  "RO": "Romania",
+  "RS": "Serbia",
+  "RU": "Russian Federation",
+  "RW": "Rwanda",
+  "SA": "Saudi Arabia",
+  "SB": "Solomon Islands",
+  "SC": "Seychelles",
+  "SD": "Sudan",
+  "SE": "Sweden",
+  "SG": "Singapore",
+  "SH": "Saint Helena, Ascension and Tristan da Cunha",
+  "SI": "Slovenia",
+  "SJ": "Svalbard and Jan Mayen",
+  "SK": "Slovakia",
+  "SL": "Sierra Leone",
+  "SM": "San Marino",
+  "SN": "Senegal",
+  "SO": "Somalia",
+  "SR": "Suriname",
+  "SS": "South Sudan",
+  "ST": "Sao Tome and Principe",
+  "SV": "El Salvador",
+  "SX": "Sint Maarten (Dutch part)",
+  "SY": "Syrian Arab Republic",
+  "SZ": "Swaziland",
+  "TC": "Turks and Caicos Islands",
+  "TD": "Chad",
+  "TF": "French Southern Territories",
+  "TG": "Togo",
+  "TH": "Thailand",
+  "TJ": "Tajikistan",
+  "TK": "Tokelau",
+  "TL": "Timor-Leste",
+  "TM": "Turkmenistan",
+  "TN": "Tunisia",
+  "TO": "Tonga",
+  "TR": "Turkey",
+  "TT": "Trinidad and Tobago",
+  "TV": "Tuvalu",
+  "TW": "Chinese Taipei",
+  "TZ": "Tanzania, United Republic of",
+  "UA": "Ukraine",
+  "UG": "Uganda",
+  "UM": "United States Minor Outlying Islands",
+  "US": "United States",
+  "UY": "Uruguay",
+  "UZ": "Uzbekistan",
+  "VA": "Holy See (Vatican City State)",
+  "VC": "Saint Vincent and the Grenadines",
+  "VE": "Venezuela, Bolivarian Republic of",
+  "VG": "Virgin Islands, British",
+  "VI": "Virgin Islands, U.S.",
+  "VN": "Viet Nam",
+  "VU": "Vanuatu",
+  "WF": "Wallis and Futuna",
+  "WS": "Samoa",
+  "YE": "Yemen",
+  "YT": "Mayotte",
+  "ZA": "South Africa",
+  "ZM": "Zambia",
+  "AA": "User defined",
+  "XZ": "International waters",
+  "QO": "Oceania",
+  "ZZ": "Unknown or invalid (used internally at GBIF for int. orgs)",
+  "TECHNICAL_POINT_OF_CONTACT": "Technical point of contact",
+  "ADMINISTRATIVE_POINT_OF_CONTACT": "Administrative point of contact",
+  "POINT_OF_CONTACT": "Point of contact",
+  "ORIGINATOR": "Originator",
+  "METADATA_AUTHOR": "Metadata author",
+  "PRINCIPAL_INVESTIGATOR": "Principal investigator",
+  "AUTHOR": "Author",
+  "CONTENT_PROVIDER": "Content provider",
+  "CUSTODIAN_STEWARD": "Custodian steward",
+  "DISTRIBUTOR": "Distributor",
+  "EDITOR": "Editor",
+  "OWNER": "Owner",
+  "PROCESSOR": "Processor",
+  "PUBLISHER": "Publisher",
+  "USER": "User",
```

Comment:
I'm thinking that the translation file is easier to read if we have nesting on for example enums. 

It is a trade-off as translators might have to translate a value twice (say USER), but it also gives us the option to have different translations for enum ContactType that has a USER and enum UserRole that also has USER. 

I personally believe that a bit more context in form of a more meaningful key, will often make it easier to translate. 

Could you consider doing `country.SO`, `country.SR`, `contactType.EDITOR`, `contactType.OWNER` etc?

Reply:
No problem. Actually, I use nested keys already for some cases.